### PR TITLE
Added 8.0 as platform #4

### PR DESCRIPTION
**Pull request by kiryteo**

### CODE REVIEW

File: podfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Uncomment this line to define a global platform for your project
-# platform :ios, '9.0'
+# platform :ios, '8.0'
```
Comment (chashmeetsingh) [inline podfile]: @kiryteo you need to uncomment this line to make it work as well.

Comment (kiryteo) [reply podfile]: Sorry, my bad.. that was the basic requirement and I left it commented. I'll change it asap.

**Comment by chashmeetsingh** [comment]
Any reason for closing the PR?

**Comment by kiryteo** [comment]
@chashmeetsingh No. a mistake.

**Pull request merged**